templates/taxi/driver_list.html

```diff
@@ -7,7 +7,7 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
In Django templates, it's recommended to use single quotes for URL tags to maintain consistency. Change `{% url "taxi:driver-detail" pk=driver.id %}` to `{% url 'taxi:driver-detail' pk=driver.id %}`.